### PR TITLE
Visitor knows when someone hear him

### PR DESCRIPTION
**Pull request by gturri**
Some code has been copy / pasted. So even if you're interested in this feature you may want to wait for some cleaning before pulling it.
However I'm creating this pull request now in case you already have some feedback.

I believe I could add a few commits to this pull request to:
- Add timestamps to message (because it's nice to see that someone is coming (or ringing), but it's nice to see if it's just now or if it happened an hour ago)
- Send feedback (like "the bell is ringing" to the client, when he clicked, to let him know if no one heard because of his 3G)
- Remove code duplication

### CODE REVIEW

File: ring.html [hunk 1]
```diff
@@ -20,11 +21,29 @@
...
     var socket = io.connect('{{url}}');
+    var inbox= document.getElementById("inbox");
     var bellButton = document.getElementById("bell");
 
     bellButton.onclick = function() {
         socket.emit('ring', { message: "Someone's here!" });
     };
+
+    // I'm a client
+    socket.emit('type', { type: 'client'});
```
Comment (AlexFaraino) [inline ring.html]: I'd prefer visitor (or guest) to client. "Host" is not here for server, it's the person who own the place, and welcomes visitors or guests :)

**Comment by AlexFaraino** [comment]
O.K. for the timestamp. That's a good idea.

About the ACK: I don't think the visitor interface should display an history of messages. Instead, can we use a simple input (like the one for monitoring connection on the /monitor page)? When someone rings, it displays "Ringing..." and when an host ACK, it displays "Someone is coming".

What do you think ?

**Comment by gturri** [comment]
"What do you think ?"
Not much: I'm not really a IHM guy.

However, you're right: a visitor isn't interested in the history. Anyway he is supposed to open the app when he arrives, and close it quite soon after.

I'll try to adapt this PR.

**Comment by gturri** [comment]
Here is another proposition.

I believe next step would be removing the "ringing" message after 10s (so that the visitor doesn't believe it's ringing non-stop)

**Comment by AlexFaraino** [comment]
I agree !

**Comment by gturri** [comment]
Here it is, here is what I add in mind regarding this feature.

Please let me know if you have any feedback, or if you'd rather have it done differently.

**Pull request merged**